app/controllers/practical_supports_controller.rb

```diff
@@ -1,5 +1,6 @@
 # Create a non-monetary assistance record for a patient.
 class PracticalSupportsController < ApplicationController
+  before_action :ensure_showing_practical_support
```

Comment:
My instinct would be that we don't need a controller-level verification and that just working on the view layer is fine, but I'm curious -- what are we trying to get out of this by checking on the controller level?

Reply:
I was probably just being overly cautious, but I was a little concerned about the fact that if you were already in the view and then practical support was turned off, you could still do updates from there.

In retrospect, it feels like too much code for not enough edge case - want me to pull it out?